packages/xo-server/src/xapi/mixins/vm.mjs

```diff
@@ -491,7 +492,12 @@ const methods = {
     hasVendorDevice: true,
 
     expNestedHvm: {
-      set: (expNestedHvm, vm) => vm.update_platform('exp-nested-hvm', expNestedHvm ? 'true' : null),
+      set: (expNestedHvm, vm) => {
+        const platformVersion = semver.satisfies(vm.hardware_platform_version, '>=3.4')
```

Comment:
can `platform_version` be `>= 3.4` in case of `XCP-ng 8.2`?

Reply:
No it will always be `3.2.1`

Reply:
3.4 is only XCP-ng 8.3 and XS 8.4